src/gui/editors/PianoRoll.cpp

```diff
@@ -4077,14 +4077,14 @@ PianoRollWindow::PianoRollWindow() :
 	QAction* drawAction = editModeGroup->addAction( embed::getIconPixmap( "edit_draw" ), tr( "Draw mode (Shift+D)" ) );
 	QAction* eraseAction = editModeGroup->addAction( embed::getIconPixmap( "edit_erase" ), tr("Erase mode (Shift+E)" ) );
 	QAction* selectAction = editModeGroup->addAction( embed::getIconPixmap( "edit_select" ), tr( "Select mode (Shift+S)" ) );
-	QAction* detuneAction = editModeGroup->addAction( embed::getIconPixmap( "automation" ), tr("Detune mode (Shift+T)" ) );
+	QAction* pitchBendAction = editModeGroup->addAction( embed::getIconPixmap( "automation" ), tr("Detune mode (Shift+T)" ) );
```

Comment:
Did you mean to change the text for this as well?

Reply:
Yeah, so the source code is clearer.

Reply:
@Anonymouqs I do not understand your reply.  Did you make a mistake and forget this line?  Will you fix it? (Hint, the text still reads `"Detune mode (Shift+T)"`